unittests/delay_tests.cpp

```diff
@@ -308,87 +419,75 @@ BOOST_AUTO_TEST_CASE(delete_auth_test) { try {
 
    BOOST_REQUIRE_EQUAL(transaction_receipt::executed, trace->receipt->status);
 
-   liquid_balance = get_currency_balance(chain, "eosio.token"_n);
-   BOOST_REQUIRE_EQUAL(asset::from_string("999900.0000 CUR"), liquid_balance);
+   chain.produce_blocks();
+
+   // Propose and approve an msig trx that transfers "quantity" tokens
```

Comment:
Prior to to this point you needed to update the permission for the active permission to add a delay to it. That seems to have gotten lost within this test.

Reply:
Thanks for your thorough review!